io/io/inc/ROOT/RFile.hxx

```diff
@@ -37,16 +37,15 @@ ROOT::RLogChannel &RFileLog();
 ## When and why should you use RFile
 
 RFile is a modern and minimalistic interface to ROOT files, both local and remote, that can be used instead of TFile
-when the following conditions are met:
-- you want a simple interface that makes it easy to do things right and hard to do things wrong;
-- you only need basic Put/Get operations and don't need the more advanced TFile/TDirectory functionalities;
-- you want more robustness and better error reporting for those operations;
-- you want clearer ownership semantics expressed through the type system rather than having objects "automagically"
-  handled for you via implicit ownership of raw pointers.
+when you only need basic Put/Get operations and don't need the more advanced TFile/TDirectory functionalities.
```

Comment:
Slightly off-topic, but I would prefer seeing this text in `RFile.cxx` or in an associated `.md` in the future. The reasoning is that documentation updates shouldn't recompile the world.

Reply:
I think ROOT wasn't doing this in the past, but it's a good argument to start doing it now ...